pkg/ocm/utils.go

```diff
@@ -11,14 +11,16 @@ import (
 )
 
 const (
-	BareMetalClusterResource string = "BareMetalCluster"
-	AMSActionCreate          string = "create"
-	AMSActionUpdate          string = "update"
-	AMSActionDelete          string = "delete"
-	CapabilityName           string = "bare_metal_installer_admin"
-	CapabilityType           string = "Account"
-	Subscription             string = "Subscription"
-	EmailDelimiter           string = "@"
+	BareMetalClusterResource   string = "BareMetalCluster"
+	AMSActionCreate            string = "create"
+	AMSActionUpdate            string = "update"
+	AMSActionDelete            string = "delete"
+	BareMetalCapabilityName    string = "bare_metal_installer_admin"
+	ArmCapabilityName          string = "arm"
```

Comment:
I think we should define the capabilities in the swagger because both BE & UI should use it, @avishayt @jkilzi WDYT?

Reply:
It's already defined in AMS, which is the source of truth.  I don't see the need to define it in the swagger as well.